assets/quick-add.css

```diff
@@ -122,6 +134,11 @@
   padding-left: 1rem;
 }
 
+.quick-add-modal__content-info--bulk-details > a:hover {
+  color: rgb(var(--color-foreground));
```

Comment:
I don't think we need to re declare the color here since it should already set to use the current color 🤔 
Maybe we can also add `text-underline-offset: 0.3rem;` though to match other places with underlined links 👍